frontend/src/hooks/useFormValidation.ts

```
@@ -0,0 +1,264 @@
+}
+
+/**
+ * Hook for managing multiple field validations
+ */
+export function useMultiFieldValidation(
+  fieldNames: string[],
+  options: UseFormValidationOptions = {},
+) {
+  const validations = fieldNames.reduce(
+    (acc, fieldName) => {
+      acc[fieldName] = useFormValidation(fieldName, options);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)